go.mod

```diff
@@ -287,7 +287,7 @@ require (
 
 require github.com/grafana/gofpdf v0.0.0-20231002120153-857cc45be447 // @grafana/sharing-squad
 
-require github.com/grafana/pyroscope/api v0.2.1 // @grafana/observability-traces-and-profiling
+require github.com/grafana/pyroscope/api v0.3.0 // @grafana/observability-traces-and-profiling
```

Comment:
ok so maybe silly question but github.com/grafana/pyroscope is at [2.0](https://github.com/grafana/pyroscope/blob/main/package.json)  so where is this version coming from?

Reply:
Haha yeah this is a bit confusing.

We publish packages from within Pyroscope which describe our public API. We do this for our API definitions ([v0.3.0](https://github.com/grafana/pyroscope/tree/api/v0.3.0)) and eBPF definitions ([v0.3.3](https://github.com/grafana/pyroscope/tree/ebpf/v0.3.3)). These packages get released on a different cycle than main Pyroscope. Since they're Go packages, we release them by pushing a tag to the Pyroscope repo to bump their version (e.g. `api/v0.3.0`).

I bumped this package here because I made some changes to the Pyroscope API to allow start/end to be handled and we needed to pull those new types into Grafana.